src/test/unit_tests/test_encryption.js

```diff
@@ -414,6 +414,10 @@ mocha.describe('Encryption tests', function() {
                 await rpc_client.account.delete_account({ email: cur_account.email});
             }));
         });
+        mocha.it('delete external connections1 succefully', async function() {
```

Comment:
Maybe a different description as it creates/deletes and not only deletes.

Reply:
thanks,  fixed.